nbs/006_diagram_base.ipynb

```diff
@@ -241,7 +241,7 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #56.                    # if node is foreach - raise an error input node can't be decision
</pre><p>There is no need to make the check in the <code>for father</code> inner loop.</p><p> </p><p>Just check for the node's input and output together in the snippet you added in line 40. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/stringdale/pull/22/discussion/#comment-2628076126'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='76'/>